src/Elasticsearch/Endpoints/Ingest/Pipeline/Get.php

```diff
@@ -23,10 +23,9 @@ class Get extends AbstractEndpoint
     public function getURI()
     {
         if (isset($this->id) !== true) {
-            throw new Exceptions\RuntimeException(
```

Comment:
It was to make this test working : https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/test/ingest/10_basic.yaml#L48

But maybe the test is wrong ?